### ISSUE TITLE
Components pages in source code

### ISSUE THREAD

**Issue by soulwasted** [creativetimofficial/argon-design-system#14]
Is it possible to get those pages (https://demos.creative-tim.com/argon-design-system/docs/foundation/colors.html) as noncompiled sources?
To create own structure, to have one SCSS source, etc.

**Comment by extrabright** [comment]
I am not sure I can understand your request? Could you rephrase and give me more details?

**Comment by soulwasted** [comment]
OK. I was just wondering if you have htmls (https://github.com/creativetimofficial/argon-design-system/tree/gh-pages/docs/components) in some kind of templates. So I can create similar Docs but with my own structure. To change left navigation in one place, not in every html.

**Comment by rarestoma** [comment]
Hi @soulwasted,

Thank you for working with our products and sorry for the late response.

Unfortunately, we do not provide the docs pages in our products.

All the best,
Rares
